eng/targets/VisualStudio.targets

```diff
@@ -69,12 +69,34 @@
 
       <_AssetsByPackageId Remove="@(_ExcludedPackageId)" />
     </ItemGroup>
+    
+    <ItemGroup>
+      <_PackageToOptimize Include="@(NuGetPackageToIncludeInVsix)" Condition="'%(NuGetPackageToIncludeInVsix.Optimization)' == 'true'" />
+      <_PackageNotToOptimize Include="@(NuGetPackageToIncludeInVsix)" Condition="'%(NuGetPackageToIncludeInVsix.Optimization)' != 'true'" />
+
+      <_AssetsToOptimize Include="@(_AssetsByPackageId)" />
+      <_AssetsToOptimize Remove="@(_PackageNotToOptimize)" />
+
+      <_AssetsNotToOptimize Include="@(_AssetsByPackageId)" />
+      <_AssetsNotToOptimize Remove="@(_PackageToOptimize)" />
+
+      <_AllAssetsToInclude Include="@(_AssetsToOptimize)">
+        <Optimization>true</Optimization>
+      </_AllAssetsToInclude>
+
+      <_AllAssetsToInclude Include="@(_AssetsNotToOptimize)">
+        <Optimization>false</Optimization>
+      </_AllAssetsToInclude>
```

Comment:
@tmat I know this looks ridicules, but it's what I could figure out with my poor MSBuild skills. There's gotta be a better way, right...?